mmdet/models/dense_heads/vfnet_head.py

```diff
@@ -411,23 +411,23 @@ def loss(self,
             bbox_weights_ini.sum()).item()
         bbox_avg_factor_ini = max(iou_targets_ini_avg_per_gpu, 1.0)
 
+        pos_decoded_bbox_preds_refine = \
+            distance2bbox(pos_points, pos_bbox_preds_refine)
+        iou_targets_rf = bbox_overlaps(
+            pos_decoded_bbox_preds_refine,
+            pos_decoded_target_preds.detach(),
+            is_aligned=True).clamp(min=1e-6)
+        bbox_weights_rf = iou_targets_rf.clone().detach()
+        iou_targets_rf_avg_per_gpu = reduce_mean(bbox_weights_rf.sum()).item()
```

Comment:
Use `reduce_mean(bbox_weights_rf.sum()).clamp_(min=1).item()` and skip `max()` operation.